packages/remix-server-runtime/CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # `@remix-run/server-runtime`
 
+## 1.8.0
+
+### Minor Changes
+
+- We have been busy at work [Layering Remix on top of React Router 6.4](https://github.com/remix-run/remix/blob/main/decisions/0007-remix-on-react-router-6-4-0.md) and are excited to be releasing the step 1 in this process that consists of running the server flow through a local copy the new framework agnostic `@remix-run/router`. ([#4612](https://github.com/remix-run/remix/pull/4612))
+- Importing functions and types from the `remix` package is deprecated, and all ([#3284](https://github.com/remix-run/remix/pull/3284))
+  exported modules will be removed in the next major release. For more details,
+  [see the release notes for 1.4.0](https://github.com/remix-run/remix/releases/tag/v1.4.0)
+  where these changes were first announced.
+- Added support for a new route `meta` API to handle arrays of tags instead of an object. For details, check out the [RFC](https://github.com/remix-run/remix/discussions/4462). ([#4610](https://github.com/remix-run/remix/pull/4610))
+
+### Patch Changes
+
+- Update to use stable `@remix-run/router@1.0.4` release ([#4720](https://github.com/remix-run/remix/pull/4720))
+- Remove duplicated router code in favor of `@remix-run/router` dependency ([#4699](https://github.com/remix-run/remix/pull/4699))
```

Comment:
Also can get rid of this one.  If we care to keep the PR references we can add them to the minor RRR bullet above